google/cloud/forseti/services/dao.py

```diff
@@ -117,7 +123,10 @@ def add_warning(self, warning):
             self.warning_store.append(warning)
 
     def get_warnings(self):
-        """Returns any stored warnings."""
+        """Returns any stored warnings.
+        Returns:
```

Comment:
ditton on new line before `Returns`; please fix this everywhere

Reply:
done